MaxText/layers/attentions.py

```diff
@@ -82,25 +83,22 @@ def apply_mask_to_logits(logits: Array, mask: Array):
   """
   return jnp.where((mask >= DEFAULT_MASK_VALUE * 0.5), logits, DEFAULT_MASK_VALUE)
 
-def _maybe_aqt_einsum(int8_training, aqt_rng):
+def _maybe_aqt_einsum(quant: Quant):
```

Comment:
Can we validate that the quantization strategy matches our expectations here? Just what we were talking about hahaha

Reply:
This change makes the code flexible to add  new strategies. Currently only added int8  but can easily add any others to quantizarion.py.